pkg/kubelet/network/exec/exec.go

```diff
@@ -29,6 +29,9 @@ limitations under the License.
 //         - setup, called after the infra container of a pod is
 //                created, but before other containers of the pod are created
 //         - teardown, called before the pod infra container is killed
+//         - status, called at regular intervals and is supposed to return a json
```

Comment:
At this point we are just following the model of how container runtime is called and then IP address is obtained from the regular Status calls.

I am expecting that at some point this Status call will include policy structures too, or custom info about the network configuration of the pod (see #12170)